### PR TITLE
Update nav link

### PR DESCRIPTION
**Pull request by zpalmtree**

### CODE REVIEW

**Comment by ExtraHash** [comment]
you need to change it everywhere

**Comment by ExtraHash** [comment]
Also all of the other links use relative paths, can we use `./` instead?

**Comment by zpalmtree** [comment]
check again

File: supply.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
   <body class="has-navbar-fixed-top">
     <nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
       <div class="navbar-brand">
-        <a class="navbar-item" href="https://turtlecoin.lol">
+        <a class="navbar-item" href="./index.html">
```
Comment (ExtraHash) [inline supply.html]: should we not maintain continuity between this and line 44

Comment (brandonlehmann) [reply supply.html]: Nope.

**Pull request merged**